spec/inputs/elasticsearch_spec.rb

```diff
@@ -756,4 +773,9 @@ def synchronize_method!(object, method_name)
     end
 
   end
+
+  # @note the hosts method got removed and @hosts i-var was changed to @seeds in elasticsearch 7.14.0
```

Comment:
Looking at the Elasticsearch client libraries, I'm seeing that [7.14.0's Transport class still has `hosts`](https://github.com/elastic/elasticsearch-ruby/blob/v7.14.0/elasticsearch-transport/lib/elasticsearch/transport/transport/base.rb#L53) (and exposes it as an attr reader -- so we shouldn't need the ivar reach-in?). What am I missing?

I _really_ would like to avoid reaching into ivars of libraries we don't own, even in specs.

Reply:
:+1: it's quite weird esp. since after https://github.com/logstash-plugins/logstash-input-elasticsearch/pull/152 things seemed :green_circle:  
but than merging with master here and :red_circle:  again ...  :disappointed: will have to revisit this once again

Reply:
probably the object we get has changed in between elasticsearch 5.x -> 7.14.0 
used to return the Transport itself but now it's a [`Transport::Client`](https://github.com/elastic/elasticsearch-ruby/blob/v7.14.0/elasticsearch-transport/lib/elasticsearch/transport/client.rb) (with an extra [`.transport`](https://github.com/elastic/elasticsearch-ruby/blob/v7.14.0/elasticsearch-transport/lib/elasticsearch/transport/client.rb#L68) to get the same as in 5.x)

Reply:
there's a work-around for this now that should be cleaner in terms of not reading i-vars.
btw. LS used elasticsearch 5.0.5 up till 7.13.4 (LS 7.14.0 upgraded to 7.5.0).